lib/std/zig/system.zig

```diff
@@ -1062,7 +1062,7 @@ fn detectAbiAndDynamicLinker(
         error.NameTooLong,
         // Finally, we fall back on the standard path.
         => |e| {
-            std.log.warn("Encountered error: {s}, falling back to default ABI and dynamic linker.\n", .{@errorName(e)});
+            std.log.warn("Encountered error: {s} whilst detecting dynamic linker in env binary, falling back to default ABI and dynamic linker.\n", .{@errorName(e)});
```

Comment:
ye olde english, forsooth I say unto thee

```suggestion
            std.log.warn("failed to detect native ABI and dynamic linker from '{s}': {s}", .{path_of_the_file_checked, @errorName(e)});
```